lib/update/cluster/phases/bootstrap.go

```diff
@@ -548,6 +576,49 @@ func (p *updatePhaseBootstrap) addUpdateRuntimePackageLabel() error {
 	return nil
 }
 
+// migrateInfluxDBData copies all from /var/lib/data/influxdb if it exist on the node and if destination
+// directory /var/lib/gravity/monitoring/influxdb empty
+func (p *updatePhaseBootstrap) migrateInfluxDBData(ctx context.Context) error {
+	newDataDirectory := "/var/lib/gravity/monitoring/influxdb"
+	oldDataDirectory := "/var/lib/data/influxdb"
+
+	if _, err := os.Stat(oldDataDirectory); os.IsNotExist(err) {
```

Comment:
I don't think this will work as expected. "oldDataDirectory" is inside the planet, right? So this will always return "not found"